cosmic-panel-bin/src/space_container/wrapper_space.rs

```diff
@@ -691,4 +691,13 @@ impl WrapperSpace for SpaceContainer {
             }
         }
     }
+
+    fn transform_changed(
+        &mut self,
+        _conn: &Connection,
+        _surface: &c_wl_surface::WlSurface,
+        _new_transform: cctk::sctk::reexports::client::protocol::wl_output::Transform,
+    ) {
+        // TODO handle the preferred transform
```

Comment:
Is it okay if this is a stub for now?

Reply:
Ya, this should be ok as a stub. My understanding of the [description](https://wayland.app/protocols/wayland#wl_surface:event:preferred_buffer_transform) is that I can indicate the transform I rendered with if I use the preferred transform, but it's ok if I don't use it.